src/js/components/Button/StyledButton.js

```diff
@@ -127,7 +127,13 @@ const StyledButton = styled.button`
   ${props =>
     !props.plain &&
     `
-    transition: 0.1s ease-in-out;
+    transition-property: color,
+      background,
```

Comment:
I'm wondering why we need `background`. I am thinking that `background-color` might be sufficient?